### DISCUSSION TITLE
Is there a way to add (YEAR) to the end of the series?

### DISCUSSION THREAD

**Discussion by mrkrstphr** [comictagger/comictagger#690] [Q&A] [answered]
I see options for renaming. The app I use to read comics will combine series of the same name, so I wanted to put the (YEAR) at the end of the series name to help the app differentiate between them.

Edit: I'm talking about the "Series" metadata information, not changing the file name.

**Comment by lordwelch** [comment]
see #680

**Reply by mrkrstphr** [reply]
Sorry, I meant I want to add it as part of the Series name in the metadata, not rename the file.

**Reply by lordwelch** [reply, marked as answer]
That will never be supported, you will have to edit them manually if you want to do that. Most apps for reading comics have their own way to support displaying the year either by using the volume or storing it in their own database

**Reply by mrkrstphr** [reply]
I don't want to display the year. The app I use for reading won't distinguish between X-Men 2021 vs X-Men 2024, for instance. It just lumps them all together as one series, even though they have different volumes in the metadata. 

I was hoping there was some kind of support for custom transformations.

Sounds like I just need to find a better app that can actually distinguish them. Thanks for your time.
